.github/workflows/build-docs.yml

```diff
@@ -0,0 +1,29 @@
+name: Docs build
+# This workflow runs when a PR is labeled with `docs`
+# This will check if the docs build successfully by running `npm run build`
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - release/*
+
+jobs:
+  check-docs-build:
+    name: Check docs build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.4
+        with:
+          persist-credentials: false
+          fetch-depth: 0
+      - uses: technote-space/get-diff-action@v5
+        id: git_diff
+        with:
+          PATTERNS: |
+            docs/*
+          SUFFIX_FILTER: |
+            .md
+      - name: Install dependencies and build docs 🧱
+        run: |
+          make build-docs
```

Comment:
Should this run the `make build-docs-versioned`?

Reply:
I think we only want to check if the build fails locally with the current branch